templates/master/00-master/azure/units/openshift-azure-routes.service.yaml

```diff
@@ -0,0 +1,11 @@
+name: openshift-azure-routes.service
+enabled: true
```

Comment:
I think this is going to start it unconditionally instead of just triggering via the `.path` unit - but would need to do some testing to verify.

Reply:
Fixed this.